spec/migrations/20170410130134_copy_server_id_from_jobs_to_miq_tasks_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_migration
+
+describe CopyServerIdFromJobsToMiqTasks do
+  let(:task_name) { "Hello Test Task" }
+  let(:task_stub) { migration_stub(:MiqTask) }
+  let(:server_id) { 1111 }
```

Comment:
Don't hardcode a server id.  You can generate one by using `ArRegion.anonymous_class_with_ar_region.rails_sequence_start`